node/rest/src/routes.rs

```diff
@@ -117,6 +128,26 @@ impl<N: Network, C: ConsensusStorage<N>, R: Routing<N>> Rest<N, C, R> {
         }
     }
 
+    // GET /<network>/status
+    pub(crate) async fn get_status(State(rest): State<Self>) -> Result<ErasedJson, RestError> {
+        // Get the CDN height (if we are connected to a CDN)
+        let cdn_height =
+            if let Some(cdn_sync) = &rest.cdn_sync { Some(cdn_sync.get_cdn_height().await?) } else { None };
```

Comment:
To prevent confusion, should we atomically determine both `cdn_height` and `sync_mode` based on `cdn_sync`? Using the `is_done()` call as you do below.